drivers/wifi/nrf700x/src/wpa_supp_if.c

```diff
@@ -1921,9 +1921,7 @@ static int nrf_wifi_vif_state_change(struct nrf_wifi_vif_ctx_zep *vif_ctx_zep,
 	vif_ctx = def_dev_ctx->vif_ctx[vif_ctx_zep->vif_idx];
 
 	vif_state_info.state = state;
-	memcpy(vif_state_info.ifacename,
-	       vif_ctx_zep->ifname,
-	       strlen(vif_ctx_zep->ifname));
+	vif_state_info.if_index = vif_ctx_zep->vif_idx;
```

Comment:
these changes are unrelated to commit log

Reply:
Included in a separate commit